### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by whoamilove** [ShiratsuyuShigure/MLAENet-pytorch#3]
Hello, your code seems to have many problems

**Comment by ShiratsuyuShigure** [comment]
Sorry for the previous version that wasn't well-organized. We have now uploaded the properly organized code.

**Comment by YZGod666** [comment]
But your code has just run the gradient explosion problem, what is the cause of this?

**Comment by ShiratsuyuShigure** [comment]
This is not gradient explosion - these are normal log messages from Apex AMP. As shown in the progress bar (black bar reaching 100%) with no further warnings appearing, the model is actually training successfully.

**Comment by YZGod666** [comment]
However, after 5 epochs, MAE minimizes and even reaches a value of 1-2, which does not match the parameters on the underlying data set in your paper
